implementations/micrometer-registry-wavefront/src/main/java/io/micrometer/wavefront/WavefrontMeterRegistry.java

```diff
@@ -54,6 +54,8 @@
 /**
  * {@link PushMeterRegistry} for Wavefront.
  *
+ * This requires Wavefront's Java SDK 2.1 or later.
```

Comment:
Turns out we need 2.2 now.
```suggestion
 * This requires Wavefront's Java SDK 2.2 or later.
```